script/DeployLeveragedEthTestnet.s.sol

```diff
@@ -1,23 +1,19 @@
 // SPDX-License-Identifier: AGPL-3.0
 pragma solidity ^0.8.13;
 
-import "forge-std/console2.sol";
-import "forge-std/Test.sol";
-
 import {DeployLeveragedEth} from "./base/DeployLeveragedEth.sol";
 import {FixedPointMathLib} from "solmate/utils/FixedPointMathLib.sol";
 import {MockWETH} from "../test/mocks/MockWETH.sol";
 import {MockUSDC} from "../test/mocks/MockUSDC.sol";
+import {GoerliConstants as C} from "../src/lib/GoerliConstants.sol";
 
 contract DeployScript is DeployLeveragedEth {
     function deployMockTokens() internal override {
-        weth = new MockWETH();
-        usdc = MockUSDC(0x30e2B7a907997fDC5a71E377Ece54FAae9F5392D);
+        weth = MockWETH(payable(C.WETH));
+        usdc = MockUSDC(C.USDC);
     }
 
     function run() external {
-        deployMockTokens();
-        deployMocks();
```

Comment:
I get why you are removing `deployMockTokens()`, but it's also needed to remove `deployMocks()`?

Reply:
because i noticed after the fact that deploy executes deployMocks also

Reply:
like after the first time i introduced this structure for local and test

Reply:
I get it. I just went to see the code. Makes sense.